File: tests/fixtures/java/HelloWorld.java
package com.example;

import java.util.List;
import java.util.ArrayList;

public class HelloWorld {

    private String name;
    private final int count;

    public HelloWorld(String name, int count) {
        this.name = name;
        this.count = count;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<String> greetAll(List<String> people) {
        List<String> greetings = new ArrayList<>();
        for (String person : people) {
            greetings.add("Hello, " + person + "!");
        }
        return greetings;
    }

    public static void main(String[] args) {
        HelloWorld hw = new HelloWorld("World", 1);
        System.out.println(hw.getName());
    }
}

File: tests/fixtures/python/HelloWorld.py
from __future__ import annotations


class HelloWorld:

    def __init__(self, name: str, count: int) -> None:
        self.name = name
        self.count = count

    def get_name(self) -> str:
        return self.name

    def set_name(self, name: str) -> None:
        self.name = name

    def greet_all(self, people: list[str]) -> list[str]:
        greetings: list[str] = []
        for person in people:
            greetings.append(f"Hello, {person}!")
        return greetings

    @staticmethod
    def main(args: list[str]) -> None:
        hw = HelloWorld("World", 1)
        print(hw.get_name())
